### PR TITLE
Update task to include new inputs

### PR DESCRIPTION
**Pull request by jepetty**
This adds our new accepted inputs to both the README and the task description so that they're recognized as valid inputs.

### CODE REVIEW

**Comment by jepetty** [comment]
@dbaeumer FYI this will fix the error message you saw! :)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ You may have specific requirements that need additional configurations beyond th
 `richNavLogOutputDirectory` | | A comma-separated list of log files created from the build when RichCodeNav.EnvVarDump is a package reference
 `sourceRootDir` | | The directory on the machine where the source code is located, if different from the default.
 `uploadRichNavArtifacts` | `false` | A boolean indicating whether Rich Nav log files should be uploaded as a build artifact. Not yet supported for GitHub Actions. Instead, please use the output variable `lspLogsPath` to publish Rich Nav artifacts.
+`configFiles` | | The config file(s) for the project(s) to be indexed either as glob patterns or specific file paths relative to the root of the repo, separated by ','
```
Comment (noellelc) [inline README.md:32]: global?

**Review by noellelc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jepetty) [reply README.md:32]: Are you referring to "glob" - that's just wildcard search pattern (like `src/**/*.tsconfig`)

**Comment by dbaeumer** [comment]
@jepetty thanks

**Pull request merged**